### PR TITLE
allow attaching files

### PR DESCRIPTION
**Pull request by plrthink**
增加点击上传功能，不过有点丑了。。。

### CODE REVIEW

File: frontend/gulpfile.coffee [hunk 1]
```diff
@@ -50,11 +50,11 @@ gulp.task 'server', () ->
     root: BUILD
     port: '8000'
 
+gulp.task 'build', ['coffee', 'stylus']
+gulp.task 'default', ['build', 'watch']
+
 gulp.task 'watch', () ->
   gulp.watch ['./app/stylus/**/*.styl'], ['stylus']
   gulp.watch ['./app/coffee/**/*.coffee', './app/coffee/**/*.js'], ['coffee']
+  gulp.watch ['../**/*.html'], ['build']
```
Comment (shanzi) [inline frontend/gulpfile.coffee:59]: @plrthink  Why did you add `*.html` here?

**Comment by shanzi** [comment]
You'd better use a hidden `<input type="file" ></input>` instead of a visible one.

Trigger the click event of the hidden input by javascript after the click of the drag-and-drop area. That will show the file panel all the same.

**Comment by shanzi** [comment]
And, please change this Pull Request to merge to develop branch instead of master :)

File: frontend/gulpfile.coffee (hunk 1 above)
Comment (plrthink) [inline frontend/gulpfile.coffee:59]: @shanzi I think I was wrong. It's truly unnecessary.

**Comment by plrthink** [comment]
@shanzi Yeah, I think your method is more elegant, I will close this PR.
